### PR TITLE
Server Settings: Add Elasticsearch static info to Cache card

### PR DESCRIPTION
**Pull request by nightnei**
Related to https://github.com/Automattic/dotcom-forge/issues/9079

## Proposed Changes
With this PR we are just adding static Elasticsearch feature indicator.

## Testing Instructions
1. Create Atomic website 
2. Open /sites -> Select atomic website -> Click on "Server settings"
3. Assert that you see the next info inside "Performance optimization" card:<br />Screenshot 2024-09-18 at 13 03 07

### CODE REVIEW

File: client/hosting/server-settings/components/cache-card/index.tsx [hunk 1]
```diff
@@ -88,102 +88,123 @@ export const CacheCard = ( {
 			headingId="cache"
 			title={ translate( 'Performance optimization' ) }
 		>
-			<div>
```
Comment (nightnei) [inline client/hosting/server-settings/components/cache-card/index.tsx]: I just removed `div`, that's why a lot of changes

Comment (fredrikekelund) [reply client/hosting/server-settings/components/cache-card/index.tsx]: Could we implement this change in https://github.com/Automattic/wp-calypso/pull/94664, too? That would make the diff easier to read

**Comment by fredrikekelund** [comment]
I mentioned in p9Jlb4-e7R-p2#comment-13372 that I think we would call more attention to this section by adding an `Elasticsearch` heading. We never resolved that, but I'll call in @matt-west here to see what he thinks? 

To clarify, I'm suggesting something like this:
Screenshot 2024-09-19 at 12 50 42

**Comment by matt-west** [comment]
@fluiddot  I had something similar in an early iteration, but I got feedback from the design team that the header was unnecessary so I dropped it.

Screenshot 2024-09-19 at 12 01 05

There’s quite a lot going on in this card already, so I don't think the header is strictly necessary. We can add it for consistency if you feel strongly about it though.

**Comment by fluiddot** [comment]
@matt-west I presume the ping from https://github.com/Automattic/wp-calypso/pull/94665#issuecomment-2360681886 was for a different person 😅, right?

**Comment by fredrikekelund** [comment]
> feedback from the design team that the header was unnecessary

In terms of purely conveying information, I agree, but in terms of getting users to notice this section, I disagree. 

> We can add it for consistency if you feel strongly about it though

I feel that it's easy to overlook that part without the heading, so my main concern is about calling attention to it. Adding the heading just seems like an easy way to do that

**Comment by matt-west** [comment]
@fluiddot Sorry, I was responding to two issues at the same time.

@fredrikekelund Let’s add the heading. 👍

File: client/hosting/server-settings/components/cache-card/index.tsx [hunk 2]
```diff
@@ -210,6 +210,34 @@ export const CacheCard = ( {
...
+						<HostingCardDescription>
+							<JetpackLogo size={ 16 } />
+							<div>
+								{ translate(
+									'Jetpack indexes the content of your site with Elasticsearch. {{a}}Learn more{{/a}}.',
+									{
+										components: {
+											a: (
+												<InlineSupportLink
+													supportContext="hosting-clear-cache"
```
Comment (fredrikekelund) [inline client/hosting/server-settings/components/cache-card/index.tsx]: We need to find a better target for this link…

**Review by fredrikekelund — COMMENTED** [review]
Looking great overall, but we need to find a more appropriate target for the support link 🤔

**Review by fredrikekelund — APPROVED** [review]
👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16883052

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @nightnei for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16883052).

**Pull request merged**